recipes/ckanbatch-deploy.rb

```diff
@@ -121,6 +121,15 @@
     group "root"
 end
 
+# Run dataset require updates notifications at 7am and 7:15am on batch
+file "/etc/cron.d/ckan-dataset-notification-due" do
+    content "00 7 * * * root /usr/local/bin/pick-job-server.sh && PASTER_PLUGIN=ckanext-data-qld #{ckan_cli} send_email_dataset_due_to_publishing_notification >/dev/null 2>&1\n"\
```

Comment:
If this gets any longer, perhaps it should be a copied cookbook file instead of inline?

Reply:
Ah, good point. It might actually be best to put something into `ckanweb-deploy-exts.rb`